deeptime/util/platform.py

```diff
@@ -15,7 +15,7 @@ def module_available(modname: str) -> bool:
     try:
         __import__(modname)
         return True
-    except ImportError:
+    except:
```

Comment:
does the builtin import function not raise an ImportError? Generally broad exception clauses can cause trouble.

Reply:
the idea was to catch more general exceptions which can occur if there's a setup problem with, e.g., pytorch: imagine pytorch is present on the system but not installed correctly (like wrong cuda version or whatever, weird things can happen), then there is no import error but something like an OSError or similar. This would effectively prohibit the use deeptime. But thinking about it I think it might be better to catch the ImportError and then have a general except clause hwihc emits a warning.

Reply:
yes, leaving the broad clause at the end emitting a warnings sounds good to me.